sites/page-details/index.json-ld

```diff
@@ -163,10 +172,10 @@
 			},
 			"notes": {
 				"en": [
-					"Although the component has been updated to version 3.1, the version 2.0 and 3.0 are still supported. However, the page content and the &lt;footer&gt; have to be located inside an &lt;article&gt; tag."
+					"Although the component has been updated to version 3.2, versions 2.0, 3.0 and 3.1 are still supported. However, the page content and the &lt;section&gt; have to be located inside a &lt;main&gt; tag."
```

Comment:
The current note is accurate, although it might be a little confusing. Here's a suggestion:
```suggestion
					"Although the component has been updated to version 3.2, versions 2.0, 3.0 and 3.1 are still supported. However, for the version 2.0, the page content and the &lt;footer&gt; have to be located inside an &lt;article&gt; tag."
```